docs/index.md

```diff
@@ -62,6 +62,12 @@ instructions if you wish to install on your own computer.
 
 When you are more comfortable with how OpenSAFELY works, we recommend following one of our OpenSAFELY walkthroughs (see [this notebook](https://github.com/opensafely/os-demo-research#opensafely-demo-materials)) to guide you through the platform workflow on your own computer with dummy data, rather than using the documentation pages alone.
 
+## How do I apply for access to OpenSAFELY?
+
+There is [guidance](apply-for-access.md) in this documentation for
+researchers who wish to request access to use OpenSAFELY with patient
+data.
```

Comment:
Maybe this could be made more concise:

```suggestion
There is [guidance](apply-for-access.md) for
researchers who wish to use OpenSAFELY with patient
data.
```